### DISCUSSION TITLE
当有固定列存在时，列宽调整时传入参数的问题

### DISCUSSION THREAD

**Discussion by feijizhadan777** [dotnetcore/BootstrapBlazor#4100] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

当存在固定列时，如果调用OnResizeColumnAsync时，传入的列名是前一个列的列名
例：
拖拽这里时：
![RDFK3YPXR4HRF}W8V_~ U9X](https://github.com/user-attachments/assets/5a2c782f-1b74-4ac0-9d8f-fbb5b784050b)
传入的OnResizeColumnAsync方法中传入的列名是"固定列"的列名;

**Comment by bb-auto** [comment]
@feijizhadan777 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@feijizhadan777 那个版本啊？

**Comment by feijizhadan777** [comment]
> @feijizhadan777 那个版本啊？

8.6.1

**Reply by ArgoZhang** [reply, marked as answer]
@feijizhadan777 升级到最新，已经修复有一段时间了
